Directory.Build.props

```diff
@@ -9,7 +9,6 @@
 		<RepositoryType>git</RepositoryType>
 		<RepositoryUrl>https://github.com/ahsanaman92/JpegXmpWritePluginMDE.git</RepositoryUrl>
```

Comment:
actually we shouldn't need to hard code this - SourceLink should be able to get the info from the git repo itself

Reply:
Done